proctor-common/src/test/java/com/indeed/proctor/common/TestProctor.java

```diff
@@ -566,6 +576,206 @@ public void testDetermineTestGroups_ForceGroupsWithDefaultToMinActive() {
                 .containsEntry("Y", testDefinitionY);
     }
 
+    @Test
+    public void testDetermineTestGroups_UnitlessAllocationMultipleTests() {
+        final TestBucket inactiveBucket = new TestBucket("inactive", -1, "");
+        final TestBucket controlBucket = new TestBucket("control", 0, "");
+        final TestBucket activeBucket = new TestBucket("active", 1, "");
+        final Allocation allocationX =
```

Comment:
nitpick: unitlessAllocation (or similar variable name) might make it easier to read the test.

Reply:
Done.